charts/catalog/templates/apiregistration-and-cert-secret.yaml

```diff
@@ -1,3 +1,8 @@
+{{- $ca := genCA "asb-ca" 3650 }}
+{{- $cn := printf "%s-catalog-apiserver" .Release.Name }}
```

Comment:
It's sort for "common name" and in the world of certificates, that abbreviation is well understood.

Reply:
Rather than ask you to just take my word for it that it's well understood...

View any (serving) certificate using a command like the following:

```
$ openssl x509 -in cert.pem -text
```

And among the output you will see something like:

```
Subject: CN=catalog-catalog-apiserver
```

Disregarding, for the moment, what it stands for, people who create or view certs are accustomed to seeing the abbreviation "cn" and understand what it means.